### DISCUSSION TITLE
deno import

### DISCUSSION THREAD

**Discussion by obrazovic** [tonaljs/tonal#311] [Q&A]
Has anyone had any luck importing tonal into deno? I've been beating my head against the wall, but can't seem to figure out the right way to do a direct web-based import (as deno documentation appears to suggest). Any ideas?

Deno 1.20.4
exit using ctrl+d or close()
> import { Tonal } from "https://cdn.jsdelivr.net/npm/@tonaljs/tonal/browser/tonal.min.js";
undefined
> console.log(Tonal)
undefined
undefined
> import { Note, Scale } from "https://cdn.jsdelivr.net/npm/@tonaljs/tonal/browser/tonal.min.js";
undefined
> console.log(Note)
undefined
undefined

**Comment by danigb** [comment]
I think this works:
```js
import * as Tonal from "https://esm.sh/@tonaljs/tonal";
```
